bin/index_db.sh

```diff
@@ -287,6 +291,21 @@ if [ "$data_only" == "true" ]; then
     echo "$CREATE_MODEL" >> create_models.job.sh
 else
   eval "$CREATE_MODEL"
+
+  if ["$train_llm" == "true"]; then
+    echo "Training LLM Weights"
```

Comment:
There is the 'train_llm.sh' script, but I figured I would keep this in here so that a model can be built by just calling one script. Should I simply remove this?